modules/ocf_kvm/facts.d/vms

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+import json
+import subprocess
+
+
+def run_command(command_args):
+    output = subprocess.check_output(command_args).decode('utf-8').split('\n')
+
+    # Return the command output split by line with any empty lines removed
+    return list(filter(None, output))
+
+
+vms_all = run_command(('/usr/bin/virsh', 'list', '--name', '--all'))
```

Comment:
`if __name__ == '__main__'`?

Reply:
Would it be a problem if the command failed to run?

Reply:
Yeah if it fails, I think the fact is just empty (or doesn't exist on the host from what I can tell when testing it). My thought here was that this would be never be imported, so it didn't matter if there was a condition here or not, but I'm fine adding it too.